incubator/hnc/e2e/tests/issues_test.go

```diff
@@ -18,16 +18,18 @@ var _ = Describe("Issues", func() {
 		nsSubSub2 = "sub-sub2"
 		nsSubChild = "sub-child"
 		nsSubSubChild = "sub-sub-child"
+		nsA = "a"
+		nsB = "b"
 	)
 
 	BeforeEach(func() {
 		cleanupNamespaces(nsParent, nsChild, nsSub1, nsSub2, nsSub1Sub1, nsSub2Sub1, 
-			nsSubSub2, nsSubChild, nsSubSubChild)
+			nsSubSub2, nsSubChild, nsSubSubChild, nsA, nsB)
 	})
 
 	AfterEach(func() {
 		cleanupNamespaces(nsParent, nsChild, nsSub1, nsSub2, nsSub1Sub1, nsSub2Sub1, 
-			nsSubSub2, nsSubChild, nsSubSubChild)
+			nsSubSub2, nsSubChild, nsSubSubChild, nsA, nsB)
```

Comment:
Is there any way to get rid of this duplication?

Reply:
I did a quick search and don't see a good way to iterate through constants with values. I was thinking of having a cleanup() function instead and have BeforeEach() and AfterEach() calling it, but then seems like I cannot create a function inside a var.

How about change the namespaces into a struct. Instead of using `nsParent` we use `ns.parent`. Then it will be easier to iterate through when cleaning up